nodeup/pkg/model/kubelet.go

```diff
@@ -146,6 +146,8 @@ func (b *KubeletBuilder) buildSystemdEnvironmentFile(kubeletConfig *kops.Kubelet
 	}
 
 	sysconfig := "DAEMON_ARGS=\"" + flags + "\"\n"
+	// Makes kubelet read /root/.docker/config.json properly
+	sysconfig = sysconfig + "HOME=\"/root" + "\"\n"
```

Comment:
Came across this with the ~/.aws directory as well - it is a pain...

Wondering if we should give kubelet etc their own directories, but if we do we can just change it when we need it.  Thanks for fixing :-)